lib/redis/hash_ring.rb

```diff
@@ -1,20 +1,21 @@
 # frozen_string_literal: true
 
-require 'zlib'
+require 'digest/md5'
 
 class Redis
   class HashRing
     POINTS_PER_SERVER = 160 # this is the default in libmemcached
 
-    attr_reader :ring, :sorted_keys, :replicas, :nodes
+    attr_reader :ring, :sorted_keys, :replicas, :nodes, :hasher
 
     # nodes is a list of objects that have a proper to_s representation.
     # replicas indicates how many virtual points should be used pr. node,
     # replicas are required to improve the distribution.
-    def initialize(nodes = [], replicas = POINTS_PER_SERVER)
+    def initialize(nodes = [], replicas = POINTS_PER_SERVER, hasher: Digest::MD5)
```

Comment:
Passing this to HashRing is a bit involved, might make sense to have a default on `Redis::Distributed`.

Reply:
Do you mean some kind of `attr_accessor` on `Redis::Distributed` class and use it in `HashRing`?

Reply:
Yup, exactly

Reply:
@byroot Done. CI is failing for sentinel, probably unrelated.